test-scripts/local-is-deployment/DeploymentPatterns/pattern-2/infra-destroy.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+echo "This is a Dummy Infrastructure destroy scripts!!\nStopping and cleaning the IS server"
```

Comment:
We can place two echo statements instead of \n. That's the convention I see in the other places of this PR as well. :)

Reply:
This is Fixed.